vocode/telephony.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import requests
+from vocode.models.agent import InformationRetrievalAgentConfig, LLMAgentConfig
+from vocode.models.telephony import CallEntity, CreateCallRequest
+from . import api_key
+
+BASE_URL = "https://0a7e-136-24-82-111.ngrok.io"
+
+
+class Telephony:
```

Comment:
i think this doesn't need to be a call, just both create_call and create_information_retrieval_call should take in a logger.

let's talk more about how this should feel from the developer perspective